imaginairy/samplers/base.py

```diff
@@ -25,18 +29,16 @@
 
 
 def get_sampler(sampler_type, model):
-    from imaginairy.samplers.ddim import DDIMSampler
-    from imaginairy.samplers.kdiff import KDiffusionSampler
-    from imaginairy.samplers.plms import PLMSSampler
 
     sampler_type = sampler_type.lower()
     if sampler_type == "plms":
         return PLMSSampler(model)
-    elif sampler_type == "ddim":
+    if sampler_type == "ddim":
         return DDIMSampler(model)
-    elif sampler_type.startswith("k_"):
+    if sampler_type.startswith("k_"):
         sampler_type = _k_sampler_type_lookup[sampler_type]
         return KDiffusionSampler(model, sampler_type)
+    raise ValueError("Invalid inputs")
```

Comment:
"invalid samplert_type"

Reply:
I assume you accidentally added a "t" to sampler